python/CMakeLists.txt

```diff
@@ -23,17 +23,25 @@ else()
     )
 endif()
 
+install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/podio_gen
+  DESTINATION ${podio_PYTHON_INSTALLDIR}
+  REGEX test_.*\\.py$ EXCLUDE  # Do not install test files
+  PATTERN __pycache__ EXCLUDE  # Or pythons caches
+  )
+
 #--- install templates ---------------------------------------------------------
 install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/templates
   DESTINATION ${podio_PYTHON_INSTALLDIR})
 
 IF (BUILD_TESTING)
   add_test( NAME pyunittest COMMAND python3 -m unittest discover -s ${PROJECT_SOURCE_DIR}/python/podio)
   PODIO_SET_TEST_ENV(pyunittest)
+  add_test( NAME pyunittest_gen COMMAND python3 -m unittest discover -s ${PROJECT_SOURCE_DIR}/python/podio_gen)
```

Comment:
`python3 -m unittest discover -s '${PROJECT_SOURCE_DIR}/python -p "test_*.py"` doesn't work to find all tests in both modules?

Reply:
Hmm that works, it seems without the `-p` it also picks up other files; I thought it would only pick up the subclassed classes